cloud/migrate-mst-cloud/entire-database.md

```diff
@@ -0,0 +1,141 @@
+Migrate smaller databases by dumping and restoring the entire database at once.
+This method works best on databases smaller than 100&nbsp;GB. For larger
+databases, consider [migrating your schema and data
+separately][migrate-separately].
+
+<highlight type="warning">
+Depending on your database size and network speed, migration can take a very
+long time. You can continue reading from your source database during this time,
+though performance could be slower. To avoid this problem, fork your database
+and migrate your data from the fork. If you write to tables in your source
+database during the migration, the new writes might not be transferred to
+Timescale Cloud. To avoid this problem, see the section on [migrating an active
+database](http://docs.timescale.com/cloud/latest/migrate-mst-cloud/#migrate-an-active-database).
+</highlight>
+
+## Prerequisites
+Before you begin, check that you have:
+*   Installed the PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
+    utilities.
+*   Installed a client for connecting to PostgreSQL. These instructions use
+    [`psql`][psql], but any client works.
+*   Created a new empty database in Timescale Cloud. For more information, see
+    the [Install Timescale Cloud section][install-timescale-cloud]. Provision
+    your database with enough space for all your data.
+*   Checked that any other PostgreSQL extensions you use are compatible with
+    Timescale Cloud. For more information, see the [list of compatible 
+    extensions][extensions]. Install your other PostgreSQL extensions.
+*   Checked that you're running the same major version of PostgreSQL on both
+    Timescale Cloud and your source database. For information about upgrading
+    PostgreSQL on your source database, see the [upgrade instructions for
+    self-hosted TimescaleDB][upgrading-postgresql-self-hosted] and [Managed
+    Service for TimescaleDB][upgrading-postgresql].
+*   Checked that you're running the same major version of TimescaleDB on both
+    Timescale Cloud and your source database. For more information, see the
+    [upgrading TimescaleDB section][upgrading-timescaledb].
+
+<highlight type="note">
+To speed up migration, compress your data. You can compress any chunks where
+data is not being currently inserted, updated, or deleted. When you finish the
+migration, you can decompress chunks as needed for normal operation. For more
+information about compression and decompression, see the
+[compression section](https://docs.timescale.com/timescaledb/latest/how-to-guides/compression/).
+</highlight>
+
+<procedure>
+
+### Migrating the entire database at once
+1.  Dump all the data from your source database into a `.bak` file, using your
+    source database connection details. If you are prompted for a password, use
+    your source database credentials:
+    ```bash
+    pg_dump -U <SOURCE_DB_USERNAME> -W \
+    -h <SOURCE_DB_HOST> -p <SOURCE_DB_PORT> -Fc -v \
+    -f <FILENAME>.bak <SOURCE_DB_NAME>
```

Comment:
```suggestion
    -f dump.bak <SOURCE_DB_NAME>
```

let's prefill what we can prefill to reduce number of steps for user
the filename is something we can define upfront